File: day3/javascript/Arithmetic.js
const prompt = require('prompt-sync')();



let correct = 0
let incorrect = 0


for (let i = 1; i <= 10; i++) {
	let x = Math.floor(Math.random()*10) ;
	let y = Math.floor(Math.random()*10) ;

	var z = x * y
	 
	const answer = prompt(`What is ${x} * ${y}?`)
	if (answer == x * y){
		correct++
		console.log("correct")
	}

	else {
		incorrect++
		console.log("incorrect")
	}


}

console.log("you have" + correct + "correct" + "answers")
	console.log("you have" + incorrect + "incorrect" + "answers")

File: day3/java/Arithmetic.java
import java.util.Scanner;
import java.util.Random;

public class Arithmetic{
	public static void main(String[] args){
	Scanner input = new Scanner(System.in);
	Random random = new Random();


int correct = 0;
int incorrect = 0;
for (int count = 0; count < 10; count++){
	int number = random.nextInt(10);
	int number2 = random.nextInt(20);	


int problem = number * number2;
System.out.println("What is" + " " + number + "*"  + number2 + "?");
int solution = input.nextInt();
if ( solution == number * number2) {
	correct ++;
	System.out.println("correct");	
}
else {
	incorrect ++;
	System.out.println("incorrect");
}


}

System.out.println("You have" + correct + "correct scores");
System.out.println("You have" + incorrect + "incorrect scores");




	}
}